src/exports/CTiglExportStep.cpp

```diff
@@ -386,8 +389,11 @@ void CTiglExportStep::ExportFusedStep(const std::string& filename)
 void CTiglExportStep::ExportShapes(const ListPNamedShape& shapes, const std::string& filename) const
 {
     STEPControl_Controller::Init();
-    Interface_Static::SetCVal("xstep.cascade.unit", "M");
-    Interface_Static::SetCVal("write.step.unit", "M");
+    // CAS_AES write step models in mm (default value)
```

Comment:
For us, these changes are problematic since we assume, that our model and all the cpacs paramters are defined in meters instead of millimeters. If you insist on millimeters, then you should provide a tigl api function, that allows setting the global unit.